ui/app/AppLayouts/CommunitiesPortal/popups/CreateCommunityPopup.qml

```diff
@@ -200,18 +200,34 @@ StatusStackModal {
                     model: fileListView.fileListModel
                     delegate: ColumnLayout {
                         width: ListView.view.width
-                        StatusCheckBox {
-                            id: fileCheckBox
+                        RowLayout {
+                            spacing: 20
                             Layout.fillWidth: true
-                            text: model.filePath
-                            font.pixelSize: 13
-                            checked: model.selected
-                            enabled: model.errorMessage === "" // TODO distinguish between error/warning
-                            onToggled: model.selected = checked
+                            StatusBaseText {
+                                Layout.fillWidth: true
+                                text: model.filePath
+                                font.pixelSize: 13
+                                elide: Text.ElideRight
+                                wrapMode: Text.WordWrap
+                                maximumLineCount: 2
+                            }
+
+                            StatusFlatRoundButton {
+                                id: removeButton
+                                width: 32
+                                height: 32
```

Comment:
This explicit `width/height` is not gonna work inside a `RowLayout`; need to use `Layout.preferredWidth/height`